src/azul/service/manifest_service.py

```diff
@@ -79,13 +79,13 @@ def __init__(self, storage_service: StorageService):
     def transform_manifest(self, format_: str, filters: Filters):
         generator = ManifestGenerator.for_format(format_, self, filters)
         object_key, presigned_url = self._fetch_cached_manifest(generator, format_, filters)
-        if not presigned_url:
+        if presigned_url is None:
             object_key = f'manifests/{uuid.uuid4()}.{generator.file_name_extension}' if not object_key else object_key
             file_name = self._generate_manifest(generator, object_key)
             presigned_url = self.storage_service.get_presigned_url(object_key, file_name=file_name)
         return presigned_url
 
-    def fetch_cached_manifest(self, format_: str, filters: Filters = None) -> Optional[str]:
+    def fetch_cached_manifest(self, format_: str, filters: Optional[Filters] = None) -> Optional[str]:
```

Comment:
I see one call site and it passes a non-None value to `filters`. So there is no need for this to be `Optional` or have a default. `for_format` would fail straight away if `filters` were `None`.